### PR TITLE
ci(npm): release

### PR DESCRIPTION
**Pull request by NicolasLopes7**
#1

### CODE REVIEW

**Review by edgarberlinck — APPROVED** [review]
LGTM

**Comment by edgarberlinck** [comment]
@NicolasLopes7  só uma coisa: É interessante usar o --freezeLockFile ao instalar as dependências.

File: .github/workflows/npm-release.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Node.js Package
```
Comment (tgmarinho) [inline .github/workflows/npm-release.yml:1]: Change the name to "Release package to NPM" for semantic reasons

**Review by tgmarinho — CHANGES_REQUESTED** [review]

File: .github/workflows/npm-release.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      
+      
+      - name: Dependency install
+        run: yarn
+      
+      - name: Compile
+        run: yarn build
```
Comment (tgmarinho) [inline .github/workflows/npm-release.yml:50]: package.json don't have build script

**Review by tgmarinho — CHANGES_REQUESTED** [review]

**Review by edumaciel10 — APPROVED** [review]

**Pull request merged**